test/it/uk/gov/hmrc/component/CaseSpec.scala

```diff
@@ -431,6 +443,38 @@ class CaseSpec extends BaseFeatureSpec {
       Json.parse(result.body) shouldBe Json.toJson(Seq(c1,c2))
     }
 
+    // currently not returning anything - as the trader name is encrypted
```

Comment:
I thought encryption was disabled locally? Shouldn’t this work?
It might hint that case insensitive search doesn’t work as planned

Reply:
yes, sorry, I meant that